### PR TITLE
#159951958 setup postgesql database

### PR DESCRIPTION
**Pull request by ruganda**
#### What does this PR do?
- Switches the database engine to PostgreSQL.
- Refactor test to use PostgreSQL
- make use of the environment variables

####  Description of Task to be completed?
The application should be able to use PostgreSQL while in development, testing and production environments

####  How should this be manually tested?
- clone the GitHub repo
- create a databases in Postgres.
- create a .env file
- open the terminal and run `source .env`
-  run `python manage.py migrate`
- run the project with `python manage.py runserver`

####  What are the relevant pivotal tracker stories?
#159951958

### CODE REVIEW

**Comment by arthuroe** [comment]
@ruganda status on this PR, why is Travis failing?

File: authors/settings.py [hunk 1]
```diff
@@ -81,8 +81,12 @@
...
 DATABASES = {
     'default': {
-        'ENGINE': 'django.db.backends.sqlite3',
-        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
+        'NAME': os.getenv('DATABASE_NAME'),
+        'USER': os.getenv('DB_USER'),
+        'PASSWORD': os.getenv('PASSWORD'),
+        'HOST': os.getenv('HOST'),
+        'PORT': '5432',
```
Comment (arthuroe) [inline authors/settings.py]: @ruganda let's store the port number as well in the `.env` file

Comment (ruganda) [reply authors/settings.py]: @arthuroe Let me work on that.

**Comment by byarustev** [comment]
LGTM

**Comment by cmplx-xyttmt** [comment]
LGTM

**Comment by marthamareal** [comment]
LGTM

File: README.md [hunk 2]
```diff
@@ -9,6 +9,31 @@ Create a community of like minded authors to foster inspiration and innovation
 by leveraging the modern web.
 
 ---
+## Setup
+
+##### Dependencies
+* [Python 3.6](https://docs.python.org/3/)
+* [Django 2.1](https://docs.djangoproject.com/en/2.1/releases/2.1/)
+* [PostgreSQL](https://www.postgresql.org/)
+* [JWT](auth0.com/docs/jwt)
```
Comment (arthuroe) [inline README.md]: Remove JWT from here.

Comment (arthuroe) [inline README.md]: @ruganda remove JWT from here.

Comment (ruganda) [reply README.md]: @arthuroe  I have removed JWT

**Pull request merged**